packages/core/content-type-builder/admin/src/components/FormModal/attributes/validation/common.js

```diff
@@ -40,7 +40,7 @@ const isNameAllowed = (reservedNames) => {
         return false;
       }
 
-      return !reservedNames.includes(value);
+      return !reservedNames.includes(toLower(value));
```

Comment:
Instead of changing the reserved attributes in the backend we could lowerCase `reversedNames` here.
That way the list of reserved name is more precise to what it is really stored.
```suggestion
      return !toLower(reservedNames).includes(toLower(value));
```

Same for usedNames

Reply:
you are right, toLower won't work for arrays. And I agree it is an overhead. 

I would like to avoid changing the `reservedNames` list just because of this use case. Maybe other people might be using that list without this use case in mind, so it could be a breaking change.

IMO, this overhead is acceptable, as it is only executed when you add an attribute (not done very often). And it's actually clear why you are lower-casing it if we put it there. 

Still, we could look other places to do this.

Reply:
Ahh, ok, it makes sense tbh, it can be misleading and can cause unexpected issues if someone else used that list elsewhere. I will take another look later today to see if we could do this in a better way, if not, I will lowercase the list in place, as you suggested

Reply:
I didn't find any other proper place to handle this, are we ok handling it in-place with a loop?

Reply:
I am ok with that 👍